src/main/java/edu/rosehulman/covidtracer/controller/EmailController.java

```diff
@@ -0,0 +1,57 @@
+package edu.rosehulman.covidtracer.controller;
+
+import edu.rosehulman.covidtracer.model.Person;
+import edu.rosehulman.covidtracer.service.EmailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+import java.util.List;
+
+@RestController
+public class EmailController {
+
+    @Autowired
+    EmailService service;
+
+
+
+    @GetMapping("/email/test")
+    public ResponseEntity<String> sendEmail() throws IOException {
+
+        service.sendTestEmail();
+        return new ResponseEntity("Email has been sent", new HttpHeaders(), HttpStatus.OK);
```

Comment:
If you tossing the angular brackets on all these methods, it makes the build warnings about using raw ResponseEntity go away.
```suggestion
        return new ResponseEntity<>("Email has been sent", new HttpHeaders(), HttpStatus.OK);
```